apps/web/modules/bookings/views/bookings-single-view.getServerSideProps.tsx

```diff
@@ -210,6 +213,16 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 
   const internalNotes = await getInternalNotePresets(eventType.team?.id ?? eventType.parent?.teamId ?? null);
 
+  if (hideOrganizerEmail) {
+    if (bookingInfo.user) {
+      bookingInfo.userDisplayEmail = maskEmail(bookingInfo.user);
+    }
+
+    bookingInfo.attendees.forEach((attendee) => {
+      attendee.email = attendee.maskedEmail ?? attendee.email;
+    });
```

Comment:
Are we sure mutating the object here is the best approach here? @Udit-takkar I'm not 100% this will have the desired effect. Can we unit test to be sure?